src/sentry/utils/json.py

```diff
@@ -142,6 +143,29 @@ def loads(
             return _default_decoder.decode(value)
 
 
+# loads JSON with `orjson` or the default function depending on `option_name`
+# TODO: remove this once we're confident that orjson is working as expected
+def loads_experimental(option_name: str, data: str | bytes, skip_trace: bool = False) -> JSONData:
+    from sentry.features.rollout import in_random_rollout
+
+    if in_random_rollout(option_name):
+        with sentry_sdk.start_span(op="sentry.utils.json.loads") if not skip_trace else nullcontext():  # type: ignore[attr-defined]
+            return orjson.loads(data)
+    else:
+        return loads(data, skip_trace)
+
+
+# dumps JSON with `orjson` or the default function depending on `option_name`
+# TODO: remove this when orjson experiment is successful
+def dumps_experimental(option_name: str, data: JSONData) -> str:
+    from sentry.features.rollout import in_random_rollout
+
+    if in_random_rollout(option_name):
+        return orjson.dumps(data).decode("utf-8")
```

Comment:
`.decode()` is the faster way to write this -- decode defaults to utf-8 since python 3

Reply:
Do you happen to know what the performance implications are of `decoding` a bytes array that is known to be in UTF-8? Can it get away with just doing a check in that case or does it have to copy stuff around?

Reply:
`bytes_object.decode()` and `bytes_object.decode('UTF-8')` do exactly the same thing -- there's just a miniscule performance overhead to the latter to build an extra arg and call vs just relying on the default